rocAL/include/meta_data/meta_node_crop_resize.h

```diff
@@ -37,7 +37,7 @@ class CropResizeMetaNode : public MetaNode {
 
    private:
     void initialize();
-    std::shared_ptr<RocalRandomCropParam> _meta_crop_param;
+    std::shared_ptr<CropParam> _meta_crop_param;
```

Comment:
Why meta data support is added only for fixed crop param? We should add support for random Crop also.

Reply:
The support is added for both. Previously this was a RocalRandomCropParam shared ptr which only supported CropResizeNode with random crop. I changed it to CropParam which is the base class for both RocalRandomCropParam and RocalCropParam so that this meta node can be used for both cases

Reply:
But from node